Makefile

```diff
@@ -490,16 +490,22 @@ git-release: package-version-to-tag changelog
 	git add "CHANGELOG.md"
 	git commit -m "Release v$(TAG)"
 	git tag "v$(TAG)"
-	git push origin "v$(TAG)"
-	git push origin "release-v$(TAG)"
 
 .PHONY: fetch-git-tags
 fetch-git-tags:
 	# Make sure we are caught up with tags
 	git fetch -t
 
-.PHONY: release
-release: fetch-git-tags release-tag-image bundle push push-index undo-deploy-tag-image git-release ## Do an official release (Requires permissions)
+.PHONY: prepare-release
+prepare-release: bundle
```

Comment:
We should still need to do release-tag-image as a prereq to bundle, instead of in release-images.

Reply:
We also need to have git-release there too. I think it would suffice for this target to be

`prepare-release: release-tag-image bundle git-release`

Reply:
but yet `git-release` makes the tag at the end... if you have to fix up a commit, you would need to replace the tag.

I suggest moving

	git commit -m "Release v$(TAG)"
	git tag "v$(TAG)"

to go above the other pushes in `push-release`

Reply:
Ok - good points. Pushing a follow up to fix these.